packages/misskey-js/src/autogen/apiClientJSDoc.ts

```diff
@@ -1,6 +1,6 @@
 /*
  * version: 2023.12.2
- * generatedAt: 2024-01-02T08:53:57.449Z
+ * generatedAt: 2024-01-02T13:57:41.524Z
```

Comment:
It seems that no actual changes have been made to the API, so it might be better not to have changes in this folder in this PR.

Reply:
Running `pnpm build-misskey-js-with-types` changed the timing of all type files. After updating the latest `develop` branch, I realized that someone else had already generated the correct types. So I'm now undoing the changes I made to the types files.